lib/AbstractMethodError.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const WebpackError = require("./WebpackError");
+const currentMethodRegexp = /at ([a-zA-Z0-9_.]*)/;
+
+/**
+ * Error for abstract method
+ * @example
+ * class FooClass {
+ *     abstractMethod() {
+ *         throw new AbstractMethodError(); // error message: Abstract method FooClass.abstractMethod. Must be overriden.
+ *     }
+ * }
+ *
+ */
+class AbstractMethodError extends WebpackError {
+	constructor() {
+		super();
+		this.name = "AbstractMethodError";
+		/** @type {RegExpMatchArray} */
+		const match = this.stack.split("\n")[1].match(currentMethodRegexp);
+		if (match && match[1]) {
+			this.message = `Abstract method ${match[1]}. Must be overriden.`;
+		}
+
+		this.details = "Abstract method was called";
```

Comment:
```suggestion
```

This doesn't add any additional information

Reply:
agree